gdnative-core/src/core_types/color.rs

```diff
@@ -47,6 +48,68 @@ impl Color {
             a: self.a + (weight * (other.a - self.a)),
         }
     }
+    #[inline]
+    pub fn blend(&self, other: &Color) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_blend)(self.sys(), other.sys()) })
+    }
+
+    #[inline]
+    pub fn contrasted(&self) -> Color {
+        Color::from_sys(unsafe { (get_api().godot_color_contrasted)(self.sys()) })
+    }
+    #[inline]
```

Comment:
Nitpick: line breaks between functions

Reply:
I'll go ahead and fix that.

Reply:
Fixed in 41969ddafffd2e4f236367481f0adb118e4c2459